app.js

```diff
@@ -3,27 +3,108 @@ const express    = require("express"),
-		{title: "Shivan's Super Scala Story", author:"Shivan", review:"Scala information", image: "https://images.unsplash.com/photo-1498050108023-c5249f4df085?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1352&q=80"},
-		{title: "Dario's Deep Database Dive", author:"Dario", review:"Databases in Depth", image: "https://images.unsplash.com/photo-1504639725590-34d0984388bd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1267&q=80"}
-		]
-	res.render("landing",{reviews:reviews});
+	res.redirect("/courses")
+})
+
+// Index - Show all courses
+app.get("/courses", (req, res) => {
+// 	Get all reviews from DB 
+	Review.find({}, (err, allReviews) => {
+		if (err) {
+			console.log(err)
+		} else {
+			res.render("landing",{reviews:allReviews});
+		}
+	});
+	
+	// New - Route to show form for new courses
+app.get("/courses/new", (req, res) => {
+		const tags = [{id: 1, title: "CSS"}, 
+					{id: 2, title: "JS"}, 
+					{id: 3, title: "NodeJS"}, 
+					{id: 4, title: "Express"}, 
+					{id: 5, title: "MongoDB"}];
+		res.render("newreview", {tags: tags});
+});
+	
+	app.get("/courses/:id", (req, res) => {
+	res.render("show");
+})
+	
+	// const reviews = [
+	// 	{title: "Noah's Normal Node Nook", author:"Noah", review:"Course about Node", image: "https://images.unsplash.com/photo-1461749280684-dccba630e2f6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80"},
+	// 	{title: "Shivan's Super Scala Story", author:"Shivan", review:"Scala information", image: "https://images.unsplash.com/photo-1498050108023-c5249f4df085?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1352&q=80"},
+	// 	{title: "Dario's Deep Database Dive", author:"Dario", review:"Databases in Depth", image: "https://images.unsplash.com/photo-1504639725590-34d0984388bd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1267&q=80"}
+	// 	]
+	// 
 });		
 
-// route for handeling logic from new review form
-app.post("/", (req, res) => {
-// 	Get data from new review form
-// 	Redirect back to the landing page
+// Create - Route to handle info from form and add a new course to DB
+app.post("/courses", (req, res) => {
+// 	Get fields from form and save in newReview variable
+	const title = req.body.title;
+	const author = req.body.author;
+	const description = req.body.description;
+	const authorUrl = req.body.authorUrl;
+	const reviewTitle = req.body.reviewTitle;
+	const reviewDetails = req.body.reviewDetails;
+	const price = req.body.price;
+	const isFree = req.body.isFree;
+	const courseUrl = req.body.courseUrl;	
+	const imageUrl = req.body.imageUrl;
+// 	Save as new var object
+	const newReview = {title: title, author: author, authorUrl: authorUrl, reviewTitle: reviewTitle, reviewDetails: reviewDetails, price: price, isFree: isFree, courseUrl: courseUrl, imageUrl: imageUrl, description: description };
+// 	Add to data base
+	Review.create(newReview, (err, newlyCreated) => {
+		if(err) {
+			console.log(err);
```

Comment:
handle errors: maybe we should have an error page we can redirect to in this case